index.html

```diff
@@ -108,9 +108,9 @@ <h3>Systers Projects</h3>
             <tr><td><a href="https://github.com/systers/vms/wiki" target="_blank">Volunteer Management System (Python w/Django)</a></td>
                 <td><a href="https://github.com/systers/portal/wiki" target="_blank">Systers Portal(Python w/Django)</a></td></tr>
```

Comment:
Minor nitpick, `<space>` required between `Portal(Python`

Reply:
I guess I have fixed everything now.

Reply:
@isunitha98selvan https://github.com/isunitha98selvan/systers.github.io/blob/c774ff5ca736f7354d2b09e92089253e078d2f40/index.html#L109, does not seem have been updated :/

Reply:
My bad. Its updated now.